gnowsys-ndf/gnowsys_ndf/es_initialize.py

```diff
@@ -0,0 +1,751 @@
+from gnowsys_ndf.ndf.models import *
+from elasticsearch import Elasticsearch
+from bson import json_util
+import os
+import sys
+import json
+reload(sys)  # Reload does the trick!
+sys.setdefaultencoding('UTF8')
+es = Elasticsearch("http://elsearch:changeit@gsearch:9200")
+
+# doc_type = ""
+author_map = {}
+group_map = {}
+author_index = "author_clix"
+index = "clix"
```

Comment:
Avoid specifying hard-coded project name like `index = "clix"`:
- gStudio is used for many projects. It's not specific to *clix*.
- This script will run on all such instances.
- Make variable `index` dynamic. There is way to import current project/site name from `settings.py`
> `from gnowsys_ndf.settings import GSTUDIO_SITE_NAME`